### PR TITLE
🥼 feat: Dynamic list of all collaborators for this repo!

### LINKED ISSUE #85: Add list of collaborators
Add a list of collaborators (translation, development, etc.) somewhere on the website, for instance in the "Who" page

### PR DESCRIPTION
**Pull request by ArnavK-09**
Closes #85

# Screen shots:;

Screenshot_2024-02-06-01-15-45-57
Screenshot_2024-02-06-01-14-12-18_40deb401b9ffe8e1df2f1cc5ba480b12

### CODE REVIEW

File: layouts/shortcodes/all-contributors.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div class="row justify-content-md-center">
+    {{ $data := getJSON "https://api.github.com/repos/Exodus-Privacy/website/contributors" }}
```
Comment (Jean-BaptisteC) [inline layouts/shortcodes/all-contributors.html:2]:
```suggestion
    {{ $data := getJSON "https://api.github.com/orgs/Exodus-Privacy/members" }}
```

or
```suggestion
    {{ $data := getJSON "https://api.github.com/repos/Exodus-Privacy/exodus/contributors" }}
```

Comment (ArnavK-09) [reply layouts/shortcodes/all-contributors.html:2]: Wait a min,

We want org members or inactive contributors?

+

Repo of website or app?

**Comment by ArnavK-09** [comment]
- [x] Fixed markdownlint

File: layouts/shortcodes/all-contributors.html (hunk 1 above)
Comment (ArnavK-09) [reply layouts/shortcodes/all-contributors.html:2]: > ```suggestion
>     {{ $data := getJSON "https://api.github.com/orgs/Exodus-Privacy/members" }}
> ```
> 
> or
> ```suggestion
>     {{ $data := getJSON "https://api.github.com/repos/Exodus-Privacy/exodus/contributors" }}
> ```

https://github.com/Exodus-Privacy/website/issues/85#issuecomment-1870970728

**Review by codeurimpulsif — CHANGES_REQUESTED** [review]
Hi, thanks but I don't think it's a good thing to add this kind of automated listing from Github (it show old contributors), also not everyone in this list want to be in the Exodus Privacy website or want to use the Github description as a description on the Exodus Privacy website. And this method delete lot of users links.

Also: I don't think we want to make a query to Github for this.

**Comment by ArnavK-09** [comment]
> Hi, thanks but I don't think it's a good thing to add this kind of automated listing from Github (it show old contributors), also not everyone in this list want to be in the Exodus Privacy website or want to use the Github description as a description on the Exodus Privacy website. And this method delete lot of users links.
> 
> Also: I don't think we want to make a query to Github for this.

Ohh mate I just tried to solve an issue #85

**Pull request closed without merging**